recipes/alteryx-open-src-update-checker/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "alteryx-open-src-update-checker" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/a/alteryx-open-src-update-checker/alteryx_open_src_update_checker-{{ version }}.tar.gz
+  sha256: 814003ab6952c406263643ea6dc9d37049ba2d53bab91775d383ebb08e39c894
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
```

Comment:
We do not have to specify the entry points since they are not `console_scripts`: 

https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=entry_points#noarch-python